content/specification.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+permalink: /specification/
+Title: Specification
+class: page-short
+---
+
+# Specification
+
+The Common Workflow Language (CWL) project produces free and open standards for describing command-line tool based workflows. These standards are [implemented](/implementations/) in many popular workflow systems. 
+
+For developers and advanced users, the current [CWL Standards v1.2.0](https://www.commonwl.org/v1.2/) provides authoritative documentation of the execution of CWL documents. The previous versions, [CWL Standards v1.0.2](https://www.commonwl.org/v1.0/) and [CWL Standards v1.1.0](https://www.commonwl.org/v1.1/), are also available.
+
```

Comment:
Needs a link to resources for non-developers or non-advanced users

Reply:
Do we have these?